web/data/accounts.go

```diff
@@ -153,6 +153,7 @@ func createAccount(c echo.Context) error {
 	}
 
 	account.Encrypt(doc)
+	account.ComputeName(doc)
```

Comment:
Should we also recompute the name when the account is updated?

Reply:
My idea was not to update the account name since the folder path depends on it when a recreation is needed. I'm not sure I fully understand all the different situations we have to handle though.